src/clr-angular/utils/drag-and-drop/drag-event-external.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {ClrDragEvent} from "./interfaces/drag-event";
+
+// This class is used to convert an internal event
+// to an external event to be emitted.
+export class ClrDragEventExt<T> {
```

Comment:
Sorry, wrong name. 😁 

The public one needs to be `ClrDragEvent`, and the private one can have any name you want.